cypress/e2e/undo-redo/add-a-block.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('adding blocks', () => {
+  beforeEach(() => {
```

Comment:
@object-required , could we extra basic command outside undo-redo folder? In fact, they are not fully undoRedo, and we may need them for other features later

Reply:
I know I created the test under it  `cypress/e2e/undo-redo/create-new-flow.cy.js` but I think we can take the opportunity to structure a bit those specs. Your thoughts?